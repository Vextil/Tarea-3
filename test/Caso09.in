Reinicio
CorrerProcesos notepad 100 vim 200 gedit 300 postgressql 400 oraclesql 500 postgis 600 eclipse 700 netbeans 800 javaVM 900 tomcat 10 jetty 20 ionic 30 geoserver 40 qgis 50 gvsig 60 firefox 70 chrome 80 gimp 90 FIN 
PedirRecursos notepad 1 vim 1 gedit 3 postgressql 4 oraclesql 4 postgis 4 javaVM 2 FIN
SolicitanOMuyConsumidores 1000 1
SolicitanOMuyConsumidores 500 1
SolicitanOMuyConsumidores 1000 4
SolicitanOMuyConsumidores 1 1
SolicitanOMuyConsumidores 301 1
SolicitanOMuyConsumidores 301 22
SolicitanOMuyConsumidores 1000 100
SolicitanOMuyConsumidores 1000 2
Reinicio
CorrerProcesos notepad 100 tomcat 10 jetty 20 ionic 30 geoserver 40 qgis 50 gvsig 60 firefox 70 chrome 80 gimp 90 FIN 
PedirRecursos notepad 1 tomcat 1 jetty 3 ionic 4 oraclesql 4 gx9 4 geoserver 4 qgis 2 firefox 80 chrome 99 FIN
SolicitanOMuyConsumidores 99 1
SolicitanOMuyConsumidores 50 1
SolicitanOMuyConsumidores 101 4
SolicitanOMuyConsumidores 1 1
SolicitanOMuyConsumidores 301 1
SolicitanOMuyConsumidores 301 22
Reinicio
CorrerProcesos notepad 100 tomcat 10 jetty 20 ionic 30 geoserver 40 FIN 
PedirRecursos notepad 1 tomcat 1 jetty 3 ionic 4 jetty 4 geoserver 4 notepad 2 notepad 80 notepad 1 tomcat 1 geoserver 3 FIN
SolicitanOMuyConsumidores 99 1
SolicitanOMuyConsumidores 101 4
SolicitanOMuyConsumidores 1 1
SolicitanOMuyConsumidores 301 1
SolicitanOMuyConsumidores 301 3
SolicitanOMuyConsumidores 301 22
Fin
