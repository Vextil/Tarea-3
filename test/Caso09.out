1>Reiniciando
2>OK
3>OK
4>notepad vim 
5>eclipse javaVM netbeans notepad postgis vim 
6>oraclesql postgis postgressql 
7>boot chrome eclipse firefox gedit geoserver gimp gvsig ionic javaVM jetty netbeans notepad oraclesql postgis postgressql qgis tomcat vim 
8>eclipse javaVM netbeans notepad oraclesql postgis postgressql vim 
9>eclipse javaVM netbeans oraclesql postgis postgressql 
10>Lista vacia. 
11>javaVM 
12>Reiniciando
13>OK
14>OK
15>boot notepad tomcat 
16>boot chrome firefox gimp gvsig notepad tomcat 
17>geoserver ionic 
18>boot chrome firefox geoserver gimp gvsig ionic jetty notepad qgis tomcat 
19>notepad tomcat 
20>Lista vacia. 
21>Reiniciando
22>OK
23>OK
24>boot notepad tomcat 
25>geoserver ionic jetty 
26>boot geoserver ionic jetty notepad tomcat 
27>notepad tomcat 
28>geoserver jetty 
29>Lista vacia. 
30>Fin
